src/support/index.coffee

```diff
@@ -105,7 +105,10 @@ module.exports = ->
     argv['prevent-browser-reload']?
 
   terminateDriver = =>
-    @driver.close()
-    @driver.quit()
+    if @driver?
+      @driver.close()
+      @driver.quit()
+    else
+      $.defer()
```

Comment:
The other way is to check the existence of driver before calling terminate. Probably cleaner.



Cheers,Tom

On Mon, Apr 27, 2015 at 10:14 PM, Sam Saccone <notifications@github.com>
wrote:

>> @@ -105,7 +105,10 @@ module.exports = ->
>>      argv['prevent-browser-reload']?
>>  
>>    terminateDriver = =>
>> -    @driver.close()
>> -    @driver.quit()
>> +    if @driver?
>> +      @driver.close()
>> +      @driver.quit()
>> +    else
>> +      $.defer()
> shouldn't we return just a resolved promise here instead of a defered object that is in a non resolved state? 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mojotech/pioneer/pull/319/files#r29192054